libs/components/forms/src/lib/modules/character-counter/character-counter-indicator.component.scss

```diff
@@ -1,11 +1,22 @@
 @use 'libs/components/theme/src/lib/styles/mixins' as mixins;
+@use 'libs/components/theme/src/lib/styles/compat-tokens-mixins' as compatMixins;
+
+@include compatMixins.sky-default-overrides('.sky-character-count-label') {
+  --sky-override-character-count-font-size: 15px;
+}
+
+@include compatMixins.sky-modern-overrides('.sky-character-count-label') {
+  --sky-override-character-count-font-size: 15px;
+}
 
 :host {
   // Force the icon and counter text to not wrap with a long form label
   flex-shrink: 0.0001;
 }
 
 .sky-character-count-label {
+  font-size: var(--sky-override-character-count-font-size, inherit);
```

Comment:
Unlike `sky-deemphasized` - the `sky-font-deemphasized` font size is set to `inhert` instead of `15px` in modern. This css is to revert that for current modern.